drone/types.go

```diff
@@ -69,6 +69,7 @@ type (
 		Version     int64  `json:"version"`
 		Signer      string `json:"signer,omitempty"`
 		Secret      string `json:"secret,omitempty"`
+		Build       Build  `json:"build,omitempty"`
```

Comment:
I am torn between whether or not we should do this:

```
type RepoBuild struct {
  Repo
  Build Build `json:"build,omitempty"`
}
```

or this:

```
type Repo struct {
  ...
  Build Build `json:"build,omitempty"`
  Builds []*Build `json:"build,omitempty"`
}
```

The repository has a one to many relationship with builds, so it feels a little weird to model a single `Build` as a child of a repository.

Reply:
I could also add `Builds`, up to you. For the incomplete API we just need the singular form.

Reply:
Just to mention it here, the core definition uses only singular: https://github.com/drone/drone/blob/master/core/repo.go#L62